streams/de.m3u

```diff
@@ -355,7 +355,7 @@ https://regiotv-s.iptv-playoutcenter.de/regiotv-s/regiotv-s.stream_1/playlist.m3
 http://wowza.telvi.de/rennsteigtv/stream_720p/playlist.m3u8
 #EXTINF:-1 tvg-id="RFH.de",RFH (1080p)
 http://58bd5b7a98e04.streamlock.net/medienasa-live/RFH_high/playlist.m3u8
-#EXTINF:-1 tvg-id="RFORosenheim.de",RFO Rosenheim (1080p)
+#EXTINF:-1 tvg-id="RFO.de",RFO Rosenheim (1080p)
```

Comment:
![image](https://github.com/iptv-org/iptv/assets/5430116/1032820b-a449-440c-accc-516da7a9105d)
Are you sure? If so name has to be changed. Also invalid certificate for the stream

Reply:
Same stream when going to https://www.rfo.de/livestream/